plugins/inputs/procstat/spec_processor.go

```diff
@@ -34,6 +35,16 @@ func NewSpecProcessor(
 	}
 }
 
+var cores int32
+
+func init() {
+	if info, err := cpu.CPUInfo(); err == nil {
```

Comment:
rather than using gopsutil, you should just use the golang builtin runtime.NumCPU(): https://golang.org/pkg/runtime/#NumCPU